src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUtilsTest.java

```diff
@@ -96,4 +96,25 @@ public void testCopyAndDeleteHDFS() throws IOException, GeneralSecurityException
         }
     }
 
+    @Test
+    public void testDirSize() throws IOException {
+        File dir = createTempDir("dir");
```

Comment:
I'm not sure that our `createTempDir()` will succeed in cleaning up a non-empty directory on exit. Can you check whether the files created by this test get cleaned up? If not, you may need to use `IOUtils.deleteRecursivelyOnExit(dir)` instead.